admin-dev/filemanager/include/utils.php

```diff
@@ -176,7 +176,7 @@ function fix_filename($str, $transliteration)
             $str = iconv('UTF-8', 'ASCII//TRANSLIT//IGNORE', $str);
         }
 
-        $str = preg_replace("/[^a-zA-Z0-9\.\[\]_| -]/", '', $str);
+        $str = preg_replace("/[^a-zA-Z0-9\\.\\[\\]_| -]/", '', $str);
```

Comment:
Can you provide example of data?
You break the `[`, `.`, `]` and the `\` escapes.